doc/sphinx/practical-tools/coqide.rst

```diff
@@ -5,130 +5,183 @@
+A menu bar and tool bar appear at the top of the window. The left-hand panel shows
+the current *script buffer*.  Each script buffer corresponds to a separate Coq process.
+The upper right panel is the *goal panel*, which shows
+the goals to be proven.  The lower right panel has three tabs: the *Messages
+panel*, which shows messages produced by commands and tactics; the *Errors panel*,
+which shows errors detected when running in :ref:`async mode <asyncmode>` and the *Jobs panel,* which shows
+information on the worker processes used by async mode.
+The contents of the right-hand panels are specific to the currently-displayed script.
+The *status bar* is a line of text that appears at the bottom of the window.
 
 Managing files and buffers, basic editing
-----------------------------------------------
-
-In the script window, you may open arbitrarily many buffers to edit.
-The *File* menu allows you to open files or create some, save them,
-print or export them into various formats. Among all these buffers,
-there is always one which is the current *running buffer*, whose name
-is displayed on a background in the *processed* color (green by default), which
-is the one where Coq commands are currently executed.
-
-Buffers may be edited as in any text editor, and classical basic
-editing commands (Copy/Paste, …) are available in the *Edit* menu.
-CoqIDE offers only basic editing commands, so if you need more complex
-editing commands, you may launch your favorite text editor on the
-current buffer, using the *Edit/External Editor* menu.
-
-Interactive navigation into Coq scripts
---------------------------------------------
-
-The running buffer is the one where navigation takes place. The toolbar offers
-five basic commands for this. The first one, represented by a down arrow icon,
-is for going forward executing one command. If that command is successful, the
-part of the script that has been executed is displayed on a background with the
-processed color. If that command fails, the error message is displayed in the
-message window, and the location of the error is emphasized by an underline in
-the error foreground color (red by default).
-
-In the figure :ref:`CoqIDE main screen <coqide_mainscreen>`,
+-----------------------------------------
+
+The *File* menu lets you open files, create new buffers, save buffers to a file,
+and print or export them in various formats.
+
+Text editing provides the basic operations such as copy, cut, paste, find and replace.
+Most editing operations are shown in the *Edit* menu.  Most menu entries show the equivalent
+keystrokes to perform the action.  If you need more complex editor commands, you can
+launch your favorite text editor on the current buffer, using the
+*Edit/External Editor* menu. When you're done editing, you must currently must
+reopen the file to see your changes.  Also note:
+
+- Undo is also available as Ctrl-Z (Command-Z on macOS).  Redo is Ctrl-Shift-Z
+  (Shift-Command-Z on macOS).
```

Comment:
MacOS shortcuts are typically abbreviated to Cmd rather than Command. The same way Control is abbreviated to Ctrl on Windows and Linux. Mac keyboards (that I have seen) have Cmd written on them.

Reply:
The Apple web page I consulted had it spelled out.  But "Cmd" is an abbreviation similar to "Ctrl" and "Alt", so more appropriate.

Reply:
Here is how Apple shortens it:

![image](https://user-images.githubusercontent.com/8614547/170846231-9faf5881-666a-4add-b947-3b978dece1e5.png)